lib/dl_connector_trino/dl_connector_trino/formula/definitions/functions_type.py

```diff
@@ -19,6 +19,80 @@
 V = TranslationVariant.make
 
 
+class FuncDbCastTrino(base.FuncDbCastBase):
+    WHITELISTS = {
+        D.TRINO: {
+            DataType.BOOLEAN: [
+                base.WhitelistTypeSpec(name="boolean", sa_type=sa.BOOLEAN),
+            ],
+            DataType.INTEGER: [
+                base.WhitelistTypeSpec(name="tinyint", sa_type=sa.SMALLINT),
+                base.WhitelistTypeSpec(name="smallint", sa_type=sa.SMALLINT),
```

Comment:
Looks like a copy-paste issue. If not, I don't understand why both types are smallint